test/unit/connectors/CustomsDataStoreConnectorSpec.scala

```diff
@@ -38,14 +38,18 @@ class CustomsDataStoreConnectorSpec extends BaseSpec {
 
   "CustomsDataStoreConnector" should {
     "successfully send a query request to customs data store and return the OK response" in {
-      when(mockHttp.doPost(any(), any(), any())(any(), any(), any())).thenReturn(Future.successful(HttpResponse(200)))
+      when(mockHttp.doPost(any(), any(), any())(any(), any(), any())).thenReturn(
+        Future.successful(HttpResponse(200, ""))
```

Comment:
Those changes to HttpResponse are to remove deprecated warning. They changed apply methods